pkg/sql/schemachanger/scplan/internal/scgraph/graph.go

```diff
@@ -110,6 +110,9 @@ func New(cs scpb.CurrentState) (*Graph, error) {
 			Attrs:    []rel.Attr{screl.ReferencedFunctionIDs},
 			Inverted: true,
 		},
+		{
+			Attrs: []rel.Attr{screl.Expr},
+		},
```

Comment:
I'm not very familiar with this part. Could you explain why this is needed?

Reply:
Ah - thank you for catching this; just bits left from testing. Excluded from my PR